tests/fixtures/basic-js/sauce-runner.json

```diff
@@ -0,0 +1,16 @@
+{
+    "playwright": {
+        "version": "1.4.1"
+    },
+    "suites": [
+        {
+            "name": "basic-js",
+            "param": {
+              "browserName": "chromium",
+              "headful": true,
+              "slowMo": 10000
+            },
+            "testMatch": "**/*.spec.js"
+        }
+    ]
+}
```

Comment:
@dpgraham should the sauce-runner.json be representing the config.yml?

Reply:
@vrunoa the sauce-runner is the result of converting config.yml to json. This JSON is the conversion of a config.yml minus unused values.